src/_tutorials/index.md

```diff
@@ -17,11 +17,16 @@ If you want a hands-on coding experience, try a
 
 ## Language tutorials
 
+The language tour shows you how to use each major Dart feature.
```

Comment:
I want to modify this intro to the language tour and tie it in better to the rest of the paragraph, but I'm not sure how yet. I'm going to play with it in a separate PR.